hcloud/version.py

```diff
@@ -1 +1,2 @@
 VERSION = '0.1.0'
+USER_AGENT = 'hcloud-python/'
```

Comment:
To me, that constant’s name is misleading since it doesn’t represent the user agent, but more like a user agent prefix (as it’s missing the version).